src/types/feature.ts

```diff
@@ -0,0 +1,17 @@
+const Feature = [
+  "featuresAssignmentsColoring",
+  "featuresAutoSaveReports",
+  "featuresDeadlineHighlighting",
+  "featuresRemoveConfirmation",
+  "featuresFilterCourses",
+  "featuresDragAndDrop",
+  "featuresReportTemplate",
+  "featuresRelativeGradesPosition",
+  "featuresDisableForceFileSaving",
+] as const
+
+type Feature = (typeof Feature)[number]
+
+const isFeature = (key: string): key is Feature => key in Feature
```

Comment:
I'm afraid `in` operator doesn't work as expected here as documented in [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in#basic_usage). It should be `Array.prototype.includes` instead.

Reply:
Thank you for your suggestion, but I'd use the `find` method instead since I somehow came across the TSC error.